### PR TITLE
Show post title in user comments sidebar

### PR DESCRIPTION
**Pull request by Reeywhaar**
#241

Made post title block in user comments sidebar. Good? Title link and link with time are the same.

screen shot 2019-01-07 at 12 31 11

### CODE REVIEW

**Comment by umputun** [comment]
I think we strip title in last comment widget ( with...), probably should do the same here

**Comment by Reeywhaar** [comment]
@umputun, fixed

screen shot 2019-01-07 at 13 07 28

File: web/app/components/comment/__title/comment__title.scss [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.comment__title {
```
Comment (igoradamenko) [inline web/app/components/comment/__title/comment__title.scss]: This code affects preview mode which is used inside last comments widget, doesn't it?

Comment (igoradamenko) [reply web/app/components/comment/__title/comment__title.scss]: I would prefer to move this elem under comment_view_user modifier. Check comment_view_preview.

File: web/app/components/comment/__title/comment__title.scss [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  margin-bottom: 0.2rem;
+
+  &-link {
```
Comment (igoradamenko) [inline web/app/components/comment/__title/comment__title.scss]: And also, don't write code in this way, please. Use full selectors. Right know it's impossible to find `__title-link` using IDE search.

Comment (Reeywhaar) [reply web/app/components/comment/__title/comment__title.scss]: ok, I get it.

File: web/app/components/comment/__title/comment__title.scss (hunk 1 above)
Comment (Reeywhaar) [reply web/app/components/comment/__title/comment__title.scss]: Oh, now I found preview mode (last comments) widget :-) Should I make the style the same as preview one?
Mine:
screen shot 2019-01-07 at 23 43 44
Same as preview mode:
screen shot 2019-01-07 at 23 44 35

Comment (Reeywhaar) [reply web/app/components/comment/__title/comment__title.scss]: @igoradamenko  check it out. Last commit can be reverted if top screenshot wins.

Comment (igoradamenko) [reply web/app/components/comment/__title/comment__title.scss]: The top screenshot looks better for me. Let's revert that two commits.

Comment (Reeywhaar) [reply web/app/components/comment/__title/comment__title.scss]: Done! Though title margin makes me doubt:

without:
screen shot 2019-01-09 at 16 53 46
with:
screen shot 2019-01-09 at 16 53 55

which is better?

Comment (igoradamenko) [reply web/app/components/comment/__title/comment__title.scss]: I don't know ¯\_(ツ)_/¯

Let's pick the second one.

Comment (Reeywhaar) [reply web/app/components/comment/__title/comment__title.scss]: Then all good :-)

**Pull request merged**